theano/compile/function_module.py

```diff
@@ -1428,6 +1429,8 @@ def create(self, input_storage=None, trustme=False):
         if self.profile:
             self.profile.linker_time += linker_time
             _fn.time_thunks = self.profile.flag_time_thunks
+            import_time = theano.gof.cmodule.import_time - start_import_time
```

Comment:
cmodule.import_time get updated in the file cmodule.py, during the function dlimport()